### PR TITLE
refactor: erda-server bootstrap openapi

### PR DESCRIPTION
**Pull request by sfwn**
#### What this PR does / why we need it:

erda-server bootstrap openapi

#### Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=325991&issueFilter__urlQuery=eyJzdGF0ZXMiOls0NDAyLDcxMDQsNzEwNSw0NDAzLDQ0MDQsNzEwNiw0NDA2LDQ0MDcsNDQxMiw0NTM4LDQ0MTMsNDQxNCw0NDE1LDQ0MTZdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=1321&type=TASK)

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  erda-server bootstrap openapi           |
| 🇨🇳 中文    |    erda-server 引导 openapi          |

### CODE REVIEW

File: erda.yml [hunk 1]
```diff
@@ -405,45 +416,6 @@ services:
...
-        mode: "container"
-    binds:
-      - "${nfs_root:/netdata}/dice-ops/dice-config/certificates/etcd-ca.pem:/certs/etcd-ca.pem:ro"
-      - "${nfs_root:/netdata}/dice-ops/dice-config/certificates/etcd-client.pem:/certs/etcd-client.pem:ro"
-      - "${nfs_root:/netdata}/dice-ops/dice-config/certificates/etcd-client-key.pem:/certs/etcd-client-key.pem:ro"
-    deployments:
-      replicas: ${replicas:1}
-      labels:
-        GROUP: "dice"
-    expose:
```
Comment (iutx) [inline erda.yml:439]: erda-server need expose port `9529`. 😄

**Review by iutx — APPROVED** [review]

File: erda.yml (hunk 1 above)
Comment (iutx) [reply erda.yml:439]:
```yaml
expose:
- 9529
```

Comment (sfwn) [reply erda.yml:439]: see line 98, already exposed using another expose syntax.

**Review by iutx — APPROVED** [review]
/lgtm

**Pull request merged**